src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double area;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.area = calculateArea();
+    }
+
+    public double getFirstLeg() {
+        return firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+        this.area = calculateArea();
+    }
+
+    public double getSecondLeg() {
+        return secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+        this.area = calculateArea();
+    }
+
+    public double getArea() {
```

Comment:
The 'getArea()' method should calculate the area each time it is called instead of returning a stored value. This ensures the area is always accurate with the current dimensions of the figure.